front_end/src/components/posts_filters.tsx

```diff
@@ -90,6 +90,10 @@ const PostsFilters: FC<Props> = ({
     []
   );
 
+  useEffect(() => {
+    setModifySearchParams(true);
+  }, []);
```

Comment:
You should add a cleanup function to this effect to reset `modifySearchParams` to `false`. Without this, the app won’t properly update the state when navigating between pages. To test, try navigating to the single question page, then to the questions feed, and back to the single question page. You’ll notice that on your second visit to the single question page, the global search will incorrectly start modifying the search parameters.

Reply:
Good point, Nikita. Makes sense, the state I set lives somewhere else. Thx! :)